bus-mapping/src/operation.rs

```diff
@@ -382,18 +382,19 @@ pub struct TxAccessListAccountOp {
     pub tx_id: usize,
     /// Account Address
     pub address: Address,
-    /// Value after the operation
-    pub value: bool,
-    /// Value before the operation
-    pub value_prev: bool,
+    /// Represents whether we can classify the access to the value as `WARM`.
+    pub is_warm: bool,
+    /// Represents whether we can classify the access to the previous value as
+    /// `WARM`.
+    pub is_warm_prev: bool,
 }
 
 impl fmt::Debug for TxAccessListAccountOp {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.write_str("TxAccessListAccountOp { ")?;
         f.write_fmt(format_args!(
             "tx_id: {:?}, addr: {:?}, val_prev: {:?}, val: {:?}",
-            self.tx_id, self.address, self.value_prev, self.value
+            self.tx_id, self.address, self.is_warm_prev, self.is_warm
```

Comment:
Please update the format string in the previous line as well!

Reply:
I forgot about them.. Apologies and good catch!!!